build-gradle-plugin/src/main/kotlin/cz/ackee/gradle/AckeePluginKotlin.kt

```diff
@@ -48,7 +48,10 @@ class AckeePluginKotlin : Plugin<Project> {
             "keystore.properties"
         )
         val keystoreProperties by project.extra(Properties().apply {
-            load(BufferedReader(FileReader(project.file(keystorePropertiesExt.fullPath))))
+            val keystorePropertiesFile = project.file(keystorePropertiesExt.fullPath)
+            if (keystorePropertiesFile.exists()) {
+                loadKeystoreProperties(keystorePropertiesFile)
```

Comment:
I was fine with the line whicih was there :D

Reply:
I can't tell if you're joking? I don't like reading nested constructors 👴

Reply:
was not joking, I really find splitting one liner to a different method with three lines unnecessary and it did not improve readability for me. You could at least reuse the method for the `appProperties` reading because it's basically the same

Reply:
Reused in 👍 b9a2a03f